shared-scripts.sh

```diff
@@ -510,3 +510,58 @@ function _integrationTest {
     export artifact_path=$CODEBUILD_SRC_DIR/../aws-amplify-cypress-api/cypress/screenshots
     mkdir -p $artifact_path && touch $artifact_path/empty.txt
 }
+
+function _uploadReportsToS3 {
+    source_version=$1
+    build_identifier=$2
+    reports_dir=$CODEBUILD_SRC_DIR/packages/amplify-e2e-tests/reports/junit
```

Comment:
for migration tests, reports_dir would be amplify-migration-tests

Reply:
Good call. I changed it now so that the package is passed in as an argument.